eng/devices/android.cake

```diff
@@ -642,10 +716,40 @@ void PrepareDevice(bool waitForBoot)
                 throw new Exception("The emulator did not finish booting in time.");
             }
 
-            if (waited % 60 == 0 && IsCIBuild())
+            // At 90 seconds, restart ADB server to recover from authorization issues
+            if (waited == 90 && IsCIBuild())
```

Comment:
Remove this timeout from this pr for now 

Let's just make this pr about the timeout on the emulator kill operation and that's it